### PR TITLE
refactor network datasource schema

### PR DESCRIPTION
**Pull request by ymaki**

### CODE REVIEW

File: ecl/data_source_ecl_network_port_v2.go [hunk 1]
```diff
@@ -14,10 +13,31 @@ func dataSourceNetworkPortV2() *schema.Resource {
...
+				Computed: true,
+			},
+			"all_fixed_ips": &schema.Schema{
+				Type:     schema.TypeList,
 				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"allowed_address_pairs": &schema.Schema{
+				Type:     schema.TypeSet,
 				Optional: true,
```
Comment (keiichi-hikita) [inline ecl/data_source_ecl_network_port_v2.go]: allowed_address_pairs can not be set as request parameter of GET according to
https://ecl.ntt.com/en/documents/api-references/network/cloud/port.html

plz remove Optional

File: ecl/data_source_ecl_network_port_v2.go [hunk 2]
```diff
@@ -34,22 +54,31 @@ func dataSourceNetworkPortV2() *schema.Resource {
...
+				},
 			},
 			"mac_address": {
 				Type:     schema.TypeString,
 				Computed: true,
 				Optional: true,
 			},
+			"managed_by_service": &schema.Schema{
+				Type: schema.TypeBool,
+				Optional: true,
```
Comment (keiichi-hikita) [inline ecl/data_source_ecl_network_port_v2.go]: Also managed by service is not request parameter as query.

File: ecl/data_source_ecl_network_port_v2.go [hunk 3]
```diff
@@ -14,10 +13,31 @@ func dataSourceNetworkPortV2() *schema.Resource {
...
+			"all_fixed_ips": &schema.Schema{
+				Type:     schema.TypeList,
 				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"allowed_address_pairs": &schema.Schema{
+				Type:     schema.TypeSet,
 				Optional: true,
+				Set:      allowedAddressPairsHash,
+				Elem: &schema.Resource{
```
Comment (keiichi-hikita) [inline ecl/data_source_ecl_network_port_v2.go]: eiichi-hikita 36 seconds ago  Member
allowed_address_pairs can not be set as request parameter of GET according to
https://ecl.ntt.com/en/documents/api-references/network/cloud/port.html

plz remove Optional.

File: website/docs/d/network_port_v2.html.markdown [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## Example Usage
+
+```hcl
+data "ecl_network_port_v2" "port_1" {
+  name = "port_1"
+}
+```
+
+## Argument Reference
+
```
Comment (keiichi-hikita) [inline website/docs/d/network_port_v2.html.markdown]: lack of "matching_subnet_cidr" in argument.

File: website/docs/d/network_subnet_v2.html.markdown [hunk 5]
```diff
@@ -57,9 +42,9 @@ The following arguments are supported:
 * `network_id` - (Required) The UUID of the parent network. Changing this
     creates a new subnet.
 
-* `ntp_servers` - (Optional) List of ntp servers.
-
-* `tags` - (Optional) Subnet tags.
+* `region` - (Optional) The region in which to obtain the V2 Neutron client.
```
Comment (keiichi-hikita) [inline website/docs/d/network_subnet_v2.html.markdown]: lack of arguments.
"subnet_id" and "status" has Optional=True, so should be appear in argument section.

File: ecl/data_source_ecl_network_port_v2.go (hunk 2 above)
Comment (ymaki) [reply ecl/data_source_ecl_network_port_v2.go]: This should be Computed: true.

File: ecl/data_source_ecl_network_port_v2.go (hunk 3 above)
Comment (ymaki) [reply ecl/data_source_ecl_network_port_v2.go]: This is also Computed: true

**Comment by ymaki** [comment]
fixed all the reviewed comments

**Pull request merged**